cmd/up/stignore.go

```diff
@@ -104,7 +104,7 @@ func addStignoreSecrets(dev *model.Dev) error {
 			model.Secret{
 				LocalPath:  transformedStignorePath,
 				RemotePath: path.Join(folder.RemotePath, ".stignore"),
-				Mode:       0600,
+				Mode:       0644,
```

Comment:
Do we need to change also the permissions of these two? https://github.com/okteto/okteto/blob/326d8b417a74580e1dfe2cbbc3360a5c46e15f0b/cmd/up/stignore.go#L173 and https://github.com/okteto/okteto/blob/326d8b417a74580e1dfe2cbbc3360a5c46e15f0b/cmd/up/stignore.go#L211

Reply:
no, those files are used as the secret so having them locally with the least permissions, the best

Reply:
I have tried to be as restrictive as possible resolving the issue. None of these filemodes were involved

Reply:
👍